source/atomics.tex

```diff
@@ -931,7 +931,8 @@
 \begin{note} The \tcode{memcpy} and \tcode{memcmp} semantics of the compare-and-exchange
 operations may result in failed comparisons for values that compare equal with
 \tcode{operator==} if the underlying type has padding bits, trap bits, or alternate
-representations of the same value.\end{note}
+representations of the same value. Notably, floating-point \tcode{-0.0} and \tcode{+0.0}
```

Comment:
I think "will" is too strong here, given that the core language rules do not require `0.0` and `-0.0` to be distinct values, and nor do they require NaNs to exist. Can you weaken this by using "may" or "will typically" or "on implementations conforming to ISO/IEC/IEEE 60559" or similar?